app/views/developers/index.html.erb

```diff
@@ -1,25 +1,25 @@
 <%= open_graph_tags title: @meta.title, turbo_native_title: t(".title"), description: @meta.description %>
```

Comment:
It looks like a lot of the classes were reordered. This is making it a bit hard to tell what was actually changed. Can you please remove the ordering changes so all that's left are additions/subtractions?

Same goes for the other HTML files.

Reply:
Sure no problem.